packages/wrangler/src/tail/index.ts

```diff
@@ -109,7 +112,7 @@ export async function createTail(
 	// send filters when we open up
 	tail.on("open", function () {
 		tail.send(
-			JSON.stringify(message),
+			JSON.stringify({ debug: debug, filters: [] }),
```

Comment:
Should we avoid sending the `filters` property at all here?
It only works because the filtering is now done before the `Session` DO and so this has no effect, since an empty array of filters will not filter out any messages, I think.
Is it that the backend still expects an array of some kind and will error otherwise?

Reply:
In the code for the `Session` DO, I can see

```ts
    if (!filters) {
      return new Response(null, { status: 204 })
    }
```

which implies all would be well if we just dropped this.